lesson2/2_8.js

```diff
@@ -7,23 +7,28 @@ const human = Object.create ({} , {
   fullName: {
     set: function (valye) { 
       [this.firstName, this.lastName] = valye.split(' ')
-      },
+      this.lastName == undefined? this.lastName = '': this.lastName 
+    },
     get () { return this.firstName + '  ' + this.lastName
```

Comment:
Вы можете безопасно хранить `undefined` для недостающих частей имени и фильтровать их при выводе:
```suggestion
    },
    get () { return [this.firstName, this.lastName].filter(Boolean).join(' ')
```